app/models/concerns/orchestration/templates.rb

```diff
@@ -0,0 +1,45 @@
+module Orchestration::Templates
+  extend ActiveSupport::Concern
+  include Orchestration::Common
+
+  included do
+    after_validation :queue_render_checks
+  end
+
+  def queue_render_checks
+    return if skip_orchestration?
+    return unless template
+
+    logger.debug "Scheduling render checks of template for #{self}"
+
+    queue.create name: _("Check renderability of '%{name}' template.") % { name: template.name },
```

Comment:
Nit: I think in English it's preferred write it like this, but looking at @Lennonka.
```suggestion
    queue.create name: _("Check renderability of template '%{name}'.") % { name: template.name },
```

Reply:
I agree with Ewoud's suggestions.